### ISSUE TITLE
The Hag

### ISSUE THREAD

**Issue by colejagdtiger** [AlexModGuy/Ice-and-Fire-Suggestions#54]
Hags would be dangerous monsters found in swap biomes, what would make them interesting is that they could be bargained with, in addition any who attempt to kill such a being could inflict curses upon them.(could add for lore that Hags taught evokers magic

Bargaining with a Hag
- Hags would not be interested in any ordinary material wealth or goods, they would what rare a valuable substances, such as dragons blood or hearts, the head of a gorgon or pixie dust, in addition they would ask for many levels of experience
- For an expensive price you could leverage a bit of a Hags own power from them allowing you to use one of their spells, however payment would need to be given again every once and a while and one must be careful not to forget as not only would they lose the power given to them but the Hag would curse them for their folly. Also it would be important to note that if you wanted to end your contract you would need to pay to do so.

Hags Spells
- Hags wield ancient and strange magic, no two Hags have the same spell array and one has no idea when going up against one what powers they may hold
- All Hags would have the power to create magical skull projectiles, these projectiles would seek there target like shulkers one do and would deal magic damage
- Hags would get 3 powers from the bellow list
- Cloud of Darkness: This would create a thick black fog that would make it impossible to see more than a few meters away
- Swarm of insects: would create a swarm of insects that would attack and have a chance to poison opponents
- Life Drain: deals a bit of magic damage and heals the Hag by whatever was taken
- ________: Inflicts the target with slowness 3 for ten seconds
- Mind Melt: Inflicts Nausea 3 for eight seconds
- Raise Undead: Creates 1-2 zombies and skeletons under the Hags control
- Summon Spirits: Creates 1-2 vexes(guys summoned by evokers)
- _______: Teleport you away from the Hag
- Death touch: Close range attack that deals lots of damage and inflicts wither
- Fear: Gives target weakness 1 for 25 seconds

Hags
- There would be three different Hag ages you would have normal Hags, grand Hags and ancient Hags, grand hags would get 4 spells from the list and ancient Hags would get 5 spell.

Curses
- If you attacked a Hag or failed to pay for the power it granted you you would be cursed. A curse could be removed by paying a Hag remove the curse(Will not do it if they cursed you and must be done by a Hag of equal or greater status to the one who cursed you) or killing the Hag who cursed you(ancient hags curses stick around even after they die so this does not work for them)
- A Hag can only and will only curse you once, however you may acquire multiple curses from multiple Hags
- A normal Hag can only grant level 1 curses a grand can grant level 2 and lower and a ancient can grant level 3 and lower

List of level 1 Curses
- Lead arm: attack speed decreased
- __________: movement speed decreased
- Curse of confusion: sometimes get brief instances of nausea 1-2

List of level 2 Curses
- Curse of Insomnia: unable to sleep
- _____: Nearby animals refuse to breed

List of level 3 Curses
- Sun Scorn: Some times are for a period of time burned by sunlight
- ___________: Max hit points reduced to 6 hearts or 12 HP

Using Hag spells
- Contracts with a normal hag only can give you 1 spell you can use, a grand hag can give you up to 2 spells to use and an ancient can give you up to 3 spells
- You have a limit on how often you can use your powers, you have a pool of points that reset every full moon or if you pay the hag to recharge them, it costs a single point every time you use a spell. A normal hag gives you 7 points to use a grand hag give you 14 and an ancient gives you 21
- In order to use them you need to have some sort of magical implement(could be a wand or a staff) this would be crafted with a wither bone and a drop of hags blood(hags drop) left clicking would use it and shift left clicking would cycle through spells

**Comment by LordDecay26** [comment]
'tis a fascinating idea that could be interesting in-game, however... i feel that it is not needed in the mod and that there are enough crazy monsters within it, making your suggestion just another stick in the pile.

**Comment by colejagdtiger** [comment]
Yeah there are a ton of monsters to add before they need to even consider thinking up new ones.

**Comment by AlexModGuy** [comment]
I like it; especially how you explained what this mob would do other than "pilis add in griffons thx1!!"

**Comment by TheLastCookie11** [comment]
nice idea colejagdtiger
